Tests/CoreDataMultipleLocalesTests.m

```diff
@@ -48,7 +48,7 @@ -(void)testMultipleLocales {
 
         LocalizedCat* nyancat = (LocalizedCat*)[self.persistenceManager fetchEntryWithIdentifier:@"nyancat"];
         XCTAssertNotNil(nyancat, @"");
-
+        XCTAssertNotNil(nyancat.picture);
```

Comment:
Can you be more precise here, as in checking on whether the `url` or the `upload` or something is set to a given value?